styles/styles.css

```diff
@@ -154,11 +154,27 @@ h4>.btn {
   margin-left: 10px;
 }
 
-.label-success {
+.label{
   float: left;
   margin-right: 9px;
 }
 
+.label-get{
+  background-color: green;
```

Comment:
Sugiro um verde mais claro.

```suggestion
  background-color: #00ae00;
```

![image](https://user-images.githubusercontent.com/6097918/59634883-252cf980-9126-11e9-9caf-eeb67500b4ed.png)